client/src/components/Splash.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+var Splash = (props) => {
+  return ( <div> Hello </div> )
```

Comment:
Try just wrapping the function with () instead of {} so you don't need to return.
```suggestion
  return ( <div> Hello </div> )
```